packages/kbn-interpreter/src/public/interpreter.js

```diff
@@ -47,6 +47,10 @@ export async function initializeInterpreter() {
   return functionList;
 }
 
+export async function isInterpreterInitialized() {
```

Comment:
A method should not start with `is` unless it returns `boolean`.  Consider `getInitializedFunctions` or returning `!!functionList`.

Reply:
On further review, since `functionList` is a Promise, `!!functionList` will always be `true`.  This method name and logic should be refactored to reflect precisely what you're trying to do... imagine if you applied TypeScript to this file, (which will happen soon, right?)